indigo_api/templatetags/indigo.py

```diff
@@ -63,12 +63,18 @@ def end_current():
 
 
 @register.simple_tag
-def commenced_provisions_description(work, commencement, uncommenced=False):
+def commenced_provisions_description(document, commencement, uncommenced=False):
+    work = document.work
     if uncommenced:
-        provisions = [p.id for p in work.uncommenced_provisions()]
+        provisions = [p.id for p in work.uncommenced_provisions(date=document.expression_date)]
     else:
         provisions = commencement.provisions
 
-    commenceable_provisions = work.commenceable_provisions()
+    commenceable_provisions = work.commenceable_provisions(date=document.expression_date)
 
     return make_beautiful(provisions, commenceable_provisions)
+
+
+@register.simple_tag
+def commencements_relevant_to_current_expression(document):
```

Comment:
Again, I think `current` is confusing, because there's no implicit current expression -- there's only the one passed in.